xcube_geodb/core/geodb.py

```diff
@@ -2461,7 +2462,8 @@ def auth_access_token(self) -> str:
     @cached_property
     def _use_winchester(self) -> bool:
         try:
-            url = self._auth_domain.replace("winchester", "")
+            p = urlparse(self._auth_domain)
+            url = f"{p.scheme}://{p.netloc}"
```

Comment:
A little hint (comment) would be good to know why this is done at all. A test would also be great.